Don't care about multi-platform support; just do enough to build nicstat.

This patch will not be sent upstream.

--- Makefile.Solaris.orig	2014-11-19 15:15:35.166755185 -0800
+++ Makefile.Solaris	2014-11-19 15:18:16.228283728 -0800
@@ -39,7 +39,6 @@
 
 
 all : $(FILES)
-	mv $? $(NATIVE_BINARY)
 
 $(NATIVE_BINARY) : $(BINARY)
 	mv $? $@
